### PR TITLE
MYST-590 terms for deb install

### PR DESCRIPTION
**Pull request by zolia**
debconf based popup to accept link to terms

### CODE REVIEW

File: bin/server_package/installation/post-uninstall.sh [hunk 1]
```diff
@@ -1,15 +1,27 @@
 #!/bin/bash
 
+if [ "$1" = "purge" -a -e /usr/share/debconf/confmodule ]; then
```
Comment (Waldz) [inline bin/server_package/installation/post-uninstall.sh]: Such silently fails are very strange

File: bin/server_package/installation/post-uninstall.sh [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+    # Source debconf library.
+    . /usr/share/debconf/confmodule
+    # Remove my changes to the db.
+    db_purge || true
```
Comment (Waldz) [inline bin/server_package/installation/post-uninstall.sh]: Same here

File: bin/server_package/installation/systemd.service [hunk 3]
```diff
@@ -11,6 +11,7 @@ Group=mysterium-node
 
 RuntimeDirectory=mysterium-node
 RuntimeDirectoryMode=0750
+LogsDirectory=mysterium-client
```
Comment (Waldz) [inline bin/server_package/installation/systemd.service]: It's `server_package...`, whould be `mysterium-client`?

File: bin/server_package/installation/post-uninstall.sh (hunk 1 above)
Comment (zolia) [reply bin/server_package/installation/post-uninstall.sh]: -a -e /usr/share/debconf/confmodule checks for existence of debconf package. And purge does not and must not fail.

**Pull request merged**